pom.xml

```diff
@@ -217,10 +217,9 @@
                             <artifactId>maven-compiler-plugin</artifactId>
                             <version>3.10.1</version>
                             <configuration>
-                                <source>${cxf.jdk.version}</source>
-                                <target>${cxf.jdk.version}</target>
                                 <maxmem>256M</maxmem>
                                 <fork>${cxf.compiler.fork}</fork>
+                                <release>11</release>
```

Comment:
Super minor, we could set it to `${cxf.jdk.version}` (it is 11)

Reply:
@reta I wrongly committed some experimental stuff. Reverted.  I should changed this with another PR and JIRA issue.